spec/models/movie_search_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+describe MovieSearch, type: :model do
+  describe 'emptysearch' do
```

Comment:
This should be either instance methods or class methods in the describe

Reply:
Thank you much.  Added a few other tests to make model test a bit more robust - this all may change as George works through the refactoring...